portfolio/src/main/webapp/index.html

```diff
@@ -10,9 +10,11 @@
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
     <script src="https://unpkg.com/aos@next/dist/aos.js"></script>
     <script src="https://www.gstatic.com/charts/loader.js"></script>
-    <script src="script.js"></script>
+		<script async defer src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCquKXcwHNhkMx8qPuRIzIsw15ixHt4HLw"></script>
+		<script type="text/javascript" src="locations.js"></script>
+		<script type="text/javascript" src="script.js"></script>
```

Comment:
Keep indexing consistent

Reply:
strange. it doesnt show up that way in the cloud shell